helpers/argumentParsers.go

```diff
@@ -621,3 +623,95 @@ func ParseBPFCmd(cmd int32) string {
 
 	return res
 }
+
+// UnameRelease gets the version string of the current running kernel
+func UnameRelease() string {
```

Comment:
You're bringing this from tracee ? I'm also using similar in BTFinfo on its new version:

https://github.com/rafaeldtinoco/libbpfgo/commit/ae3c034dadf449ef36b7a6701520522db50473fc

 Maybe we should put this in a common file to all helpers.

Reply:
yeah, i took it from tracee. 
i agree - this should be a common helper in libbfgo, and tracee should consume that wherever needed.

Reply:
Agree. We should move both UnameRelease() and kernelIsAtLeast() to helpers

Reply:
should it be as part of this PR?

Reply:
I vote for a simple PR doing the needed moves first and then this one rebased on top of that. That because I'm also using Uname function in my new OSInfo class (renaming BTFInfo class). I can also rebase on top of this new PR.